src/cpp/shared/expansions.h

```diff
@@ -12,7 +12,26 @@ class combatExtra {
         stackExtra stack;
 };
 
+class campaignExtra {
```

Comment:
This is a brand new class, not present in the original game, right?

Reply:
Yes. It's a new one